docs/csharp/methods.md

```diff
@@ -34,7 +34,7 @@ This topic contains the following sections:
 - [Expression-bodied members](#expr)
 - [Iterators](#iterators)
 
-<a name="signatures" /a>
+<a name="signatures" />
```

Comment:
You can use @GuardRex suggestion. It doesn't matter. As long as the whole section is not rendering as a link.